cmd/dapr.go

```diff
@@ -18,7 +18,7 @@ import (
 
 var RootCmd = &cobra.Command{
 	Use:   "dapr",
-	Short: "Dapr CLI",
+	Short: "Dapr Client",
```

Comment:
Do not understand this change

Reply:
Spelling out CLI.

Reply:
But CLI is Command Line Interface not Client. So I do not think this has to change. Certainly should not be Client

Reply:
Fixing it.